seq2science/cli.py

```diff
@@ -201,6 +203,38 @@ def _init(args, workflows_dir, config_path):
             shutil.copyfile(src, dest)
 
 
+def subjectively_prettier_error(arg, message):
+    """raise an exception and catch it for a subjectively prettier message"""
+    try:
+        raise argparse.ArgumentError(arg, message)
+    except argparse.ArgumentError as err:
+        print(f"\n{err}")
+        sys.exit(1)
+
+
+def add_profile_args(profile_file, parsed_args):
+    """read profile and add new arguments to parsed args"""
+    profile = yaml.safe_load(open(profile_file).read())
+
+    # parse yaml dicts
+    for k, v in profile.items():
+        if isinstance(v, list) and "=" in v[0]:
+            profile[k] = {}
+            for e in v:
+                k2, v2 = e.split("=")
+                profile[k][k2] = v2
```

Comment:
This section converts a YAML list with `=` in the elements to a dict. Without this section, the code works as intended with proper profile YAMLs.